geode-assembly/build.gradle

```diff
@@ -524,6 +523,7 @@ distributions {
         from { project(':extensions:geode-modules-assembly').distTcServer30 }
         from { project(':extensions:geode-modules-assembly').distTomcat }
         from { project(':extensions:geode-modules-assembly').distAppServer }
+        from { project(':geode-for-redis').distZip }
```

Comment:
Being a *nix user, I prefer the archives be in tgz format. Thoughts?

Reply:
Well, those other jobs just above this one like `distTomcat`, `distAppServer` etc. produce zip files. So I was trying to match the archive format of all of the other stuff in this modules directory. Make sense?